velox/exec/tests/utils/SerializedPageUtil.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/exec/tests/utils/SerializedPageUtil.h"
+
+using namespace facebook::velox;
+
+namespace facebook::velox::exec::test {
+
+std::unique_ptr<SerializedPage> toSerializedPage(
+    const RowVectorPtr& vector,
+    memory::MemoryPool* pool,
+    std::shared_ptr<OutputBufferManager> bufferManager) {
```

Comment:
Should bufferManager be const& ?

Reply:
Updated, thanks @aditi-pandit. Could you please take another look?